### PR TITLE
Inskin Bid Adapter: override schain with publisher id

### PR DESCRIPTION
**Pull request by cciocov**
## Type of change

- [ ] Bugfix
- [x] Feature
- [ ] New bidder adapter   
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Does this change affect user-facing APIs or examples documented on http://prebid.org?
- [ ] Other

## Description of change
We are sending a new parameter when making the ad call to our ad server.

### CODE REVIEW

**Comment by patmmccann** [comment]
You want to override the supply chain objects publishers are setting in config?

**Comment by ncolletti** [comment]
Is the correct place for schain data in `source.ext.schain`? Also, should you be choosing to pass the `schain` object from `validBidRequests` first, before looking to write your own from user defined adapter params?

**Comment by cciocov** [comment]
@patmmccann for the moment, we are not taking into account the schain config set in prebid, we are only concerned with passing what's defined in publisherId in our adapter config.

@ncolletti at the moment we only want to pass our own defined settings, we'll consider the rest in a future update, thanks!

File: test/spec/modules/inskinBidAdapter_spec.js [hunk 1]
```diff
@@ -12,7 +12,8 @@ const REQUEST = {
     'bidder': 'inskin',
     'params': {
       'networkId': 9874,
-      'siteId': 730181
+      'siteId': 730181,
+      'publisherId': 123456
```
Comment (ncolletti) [inline test/spec/modules/inskinBidAdapter_spec.js:16]: Could you include this param in your markdown file

Comment (cciocov) [reply test/spec/modules/inskinBidAdapter_spec.js:16]: @ncolletti This has been done, thanks!

**Review by ncolletti — APPROVED** [review]

**Pull request merged**